xml/ha_storage_protection.xml

```diff
@@ -809,10 +809,53 @@ Timeout (msgwait)  : 10
      </callout>
    </calloutlist>
   </step>
-  <step>
-   <para>For a two-node cluster, decide if you want predictable or random delays. For
+  <step xml:id="st-ha-storage-protect-fencing-static-random">
+   <para>For a <emphasis>two-node cluster</emphasis>, decide if you want to set the following delays. For
    other cluster setups you do not need to set this parameter.</para>
    <variablelist>
+    <varlistentry>
+     <term>Priority fencing delays</term>
+     <listitem>
+       <para>
+        This parameter delays nodes with the same priority (by default disabled).
+        Users set the priority of a resource which are more important to them.
+        A promoted resource has a higher priority than a started or a stopped one.
+        For a node, the priority is the sum of all priorities of resources running
+        on them.
+        <!--
+        The current solution for a two-node cluster uses the random/static
+        fencing delays described in the following items. However, to prevent
+        accidental double-fencing, a <parameter>priority</parameter> meta-attribute
+        on a node is taken into account. The priority of a node is the sum
+        of all priorities of resources running on them.-->
+      </para>
+      <para>
+       The current solution for a two-node cluster uses the random/static
+       fencing delays (described in the next items). However, this could lead to
+       accidental double-fencing.
+       With the priority fencing delays, priorities are taken into account and
+       avoid such situations.
```

Comment:
That doesn't make sense: _With the priority fencing delays, [...] and avoid such situations._ You can't say _taken into account_ followed by _and avoid_.

Reply:
Right, I'll remove the last part with "and avoid...".